src/Cms/App.php

```diff
@@ -1499,7 +1499,6 @@ public function site(): Site
 		return $this->site ??= new Site([
 			'errorPageId' => $this->options['error'] ?? 'error',
 			'homePageId'  => $this->options['home']  ?? 'home',
-			'kirby'       => $this,
```

Comment:
What's the difference with following line?
https://github.com/getkirby/kirby/blob/3.9.8/src/Cms/App.php#L1554

Reply:
No difference. I just removed it from there as well.